docs/modules/traits/pages/tracing.adoc

```diff
@@ -1,5 +1,7 @@
 = Tracing Trait
 
+// Start of autogenerated code - DO NOT EDIT! (badges)
+// End of autogenerated code - DO NOT EDIT! (badges)
 // Start of autogenerated code - DO NOT EDIT! (description)
 WARNING: The Tracing trait has been **deprecated** in favor of the xref:traits:telemetry.adoc[Telemetry] trait.
```

Comment:
Probably you want to add this trait as well

Reply:
Like so `+camel-k:deprecated=2.2.0`